src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/BaselineHelper.cs

```diff
@@ -63,6 +63,26 @@ public static string DiffFiles(string file1Path, string file2Path, ITestOutputHe
             return diffResult.StdOut;
         }
 
-        private static string GetBaselineFilePath(string baselineFileName) => Path.Combine(Directory.GetCurrentDirectory(), "baselines", baselineFileName);
+        public static string GetAssetsDirectory() => Path.Combine(Directory.GetCurrentDirectory(), "assets");
+
+        private static string GetBaselineFilePath(string baselineFileName) => Path.Combine(GetAssetsDirectory(), "baselines", baselineFileName);
+
+        private static string? CompareFilesInternal(string baselineFilePath, string actualFilePath, ITestOutputHelper outputHelper)
```

Comment:
@MichaelSimons I don't like this method name, so if you have any other ideas for it let me know.

Reply:
Is a separate method necessary here?  Why not make this the CompareFiles implementation with the addition of the Assert.  Then CompareContents can just call it and there is no need for it to have an Assert.  This would eliminate the naming concerns